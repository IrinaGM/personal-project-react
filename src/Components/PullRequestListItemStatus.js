import React from "react";

const PullRequestListItemStatus = () => {};

export default PullRequestListItemStatus;