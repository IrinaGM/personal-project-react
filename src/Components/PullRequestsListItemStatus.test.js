import React from "react";
import ShallowRenderer from 'react-test-renderer/shallow';

import PullRequestListItemStatus from "./PullRequestListItemStatus";

describe('Testing PullRequestListItemStatus', () => {
    test('testing closed status', () => {
        const renderer = new ShallowRenderer();
        const element = renderer.render(<PullRequestListItemStatus status="closed"/>);
        expect(element).toMatchSnapshot();
    });

    test('testing open status', () => {
        const renderer = new ShallowRenderer();
        const element = renderer.render(<PullRequestListItemStatus status="open"/>);
        expect(element).toMatchSnapshot();
    });
})