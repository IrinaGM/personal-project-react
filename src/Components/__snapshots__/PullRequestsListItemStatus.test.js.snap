// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing PullRequestListItemStatus testing closed status 1`] = `
<ForwardRef
  pill={true}
  variant="primary"
>
  closed
</ForwardRef>
`;

exports[`Testing PullRequestListItemStatus testing open status 1`] = `
<ForwardRef
  pill={true}
  variant="success"
>
  open
</ForwardRef>
`;
